project_0a/project_0a_3.c

```diff
@@ -0,0 +1,30 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 3 - build a C to F converter. F = C × 1.8 + 32 that goes from 0 to 100 C, 1 by 1.
+*/
+
+#include <stdio.h>
+
+int main() {
+  // Convert C to F
+  // F = C × 1.8 + 32
+
+  float fahrenheit;
```

Comment:
Local variables should be declared as close to first use as possible. Initilization is prefered over declaration and assignment. They should be declared in the narrowest scope possible. https://google.github.io/styleguide/cppguide.html#Local_Variables